manifests/istio-telemetry/grafana/dashboards/istio-performance-dashboard.json

```diff
@@ -976,7 +976,7 @@
           "step": 2
         },
         {
-          "expr": "sum(container_memory_usage_bytes{job=\"kubernetes-cadvisor\",container=~\"discovery|istio-proxy\", pod=~\"istio-pilot-.*\"})",
+          "expr": "sum(container_memory_usage_bytes{job=\"kubernetes-cadvisor\",container=~\"discovery|istio-proxy\", pod=~\"istiod-.*\"})",
```

Comment:
these changes aren't backwards compatible. should we use an inclusive expression here (and below)?

Reply:
That would be `pod=~\"istiod-.*|istio-pilot-.*\"`, right? sounds good to me

Reply:
I have made this change.